### DISCUSSION TITLE
[5.x/6.0] [UX] Review default ordering settings for the core backend lists

### DISCUSSION THREAD

**Discussion by zero-24** [joomla/joomla-cms#47042] [2. Cold Features]
### Problem identified

When opening the extensions list right now the ordering is by NAME asc while most of the time you want to edit the last installed extension right so ID decending makes more sense?
Similiar applies to com_plugins where right now its orderd by type but you would like to open the latest installed most of the time right? And if not you need to search for the plugin name anyway right?

### Proposed solution

Change the default ordering from NAME asc to ID decending (com_installer)
Change the default ordering from TYPE asc to ID decending (com_plugin)

### Open questions

The change itself is trivial but I would like to hear what others think on the Idea and where this improvment should be send against? 5.2-dev or 6.0-dev?

**Comment by chmst** [comment]
I think that this is only one of many places where the default ordering of lists is not as users expect. 
So would like to have a generic solution for the whole backend.

**Comment by tkuschel** [comment]
We could think about it -- to implement not a default ordering, but to store the last set ordering (or to enable to store a default ordering) at the backend. Next visit, the ordering of a table is like the last one.
Not everyone want to search for the latest ID first, etc.

**Comment by HLeithner** [comment]
Ordering by title ASC is the expected behavior, sorting by ID DESC would nobody expect.

**Comment by brianteeman** [comment]
> Ordering by title ASC is the expected behavior, sorting by ID DESC would nobody expect.

id desc is the default for articles

https://github.com/joomla/joomla-cms/blob/200ca9c11d57ddaab3ef302517b05434dd38d86a/administrator/components/com_content/src/Model/ArticlesModel.php#L521-L534



This is already in place

**Comment by chmst** [comment]
My users want ID DESC for articles. Because it is much more probable that they need to fix an error in the newest article. TITLE DESC is useless in most cases where hundreds of items are in the list.

**Comment by brianteeman** [comment]
> My users want ID DESC for articles. Because it is much more probable that they need to fix an error in the newest article. TITLE DESC is useless in most cases where hundreds of items are in the list.

which is why it is the default

**Comment by HLeithner** [comment]
ok might be my wrong assumptions. Having a full configurable system for personal preferences would be my favored solution. Our list view need much love on all ends and needs to be more comfortable for developer and users. Anyway I'm out form the order discussion. if you like to change the order then please create a pr.

**Comment by Fedik** [comment]
I think ordering by name for plugins and extensions is good.

**Comment by carlitorweb** [comment]
For plugins, by name is okay. Most of the time you will search for the plugin, instead of go trought the list.

For the extension list, by ID is okay. Most of the time when you go to there is for unisntall a extension, and the greatest probability is the last extensions that you have installed

**Comment by brianteeman** [comment]
The comments here show that there is no one solution fits all
